README.rst

```diff
@@ -279,15 +273,12 @@ See `Migrating content to a different collection <https://docs.ansible.com/ansib
 Q: How can I fix bugs in Ansible 2.9?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The `previous policy <https://docs.ansible.com/ansible/latest/community/development_process.html#making-your-pr-merge-worthy>`_ was:
+Please note that Ansible 2.9 no longer receives bugfixes. Only security issues can be fixed, and eventually it will be end of line with no more fixes accepted. See `ansible-core release cycle <https://docs.ansible.com/ansible/devel/reference_appendices/release_and_maintenance.html#ansible-core-release-cycle>`_ for whether 2.9 is still accepting security fixes or not.
```

Comment:
End of line -> End of life ?

Reply:
```suggestion
Please note that Ansible 2.9 no longer receives bugfixes. Only security issues can be fixed, and eventually it will be end of life with no more fixes accepted. See `ansible-core release cycle <https://docs.ansible.com/ansible/devel/reference_appendices/release_and_maintenance.html#ansible-core-release-cycle>`_ for whether 2.9 is still accepting security fixes or not.
```

Reply:
Thanks, that's what it was supposed to be :)